### DISCUSSION TITLE
Use of mathjs cells

### DISCUSSION THREAD

**Discussion by dvd101x** [gzuidhof/starboard-notebook#107] [Q&A] [answered]
Hi I just found about this project and some of it’s ideas seem genius to me.

One very useful tool I use for math calculations is mathjs. It has an expression parser for math calculations that is very powerful. Is there a way to create new cell types? Let’s say that additional to markdown, JavaScript, etc. I have like a mathjs type? Something similar to https://mathnotepad.com/

I have been trying to use mathjs from a starboard notebook but haven’t had success yet as it’s not a module I guess.

**Comment by unhott** [comment, marked as answer]
Yes, you can create custom cell types. They can be created in notebook or via plugins :)

Check out some of the info from this closed issue thread

https://github.com/gzuidhof/starboard-notebook/issues/72

**Reply by dvd101x** [reply]
Thank you so much.

**Reply by unhott** [reply]
Glad I could help!
